### ISSUE TITLE
Updating the global tests namelist and configuration file format

### ISSUE THREAD

**Issue by junwang-noaa** [ufs-community/ufs-weather-model#1663]
## Description
This is task 1 listed in issue #1248 (UFS WM regression test namelist template updates). The nemalist templates and configuration files (model_configure and nems.configure) used in the global RT tests will be cleaned up and updated to have consistent format as the global workflow. This goal is to remove the discrepancy between RT test runs and the global workflow runs.  

## Solution
Update global test namelist templates, model_configure and nems.configure files to make sure ufs wm RT test and global workflow have same format configuration files. Will work with global workflow team as the global workflow may also need to be updated.

**Comment by junwang-noaa** [comment]
@SadeghTabas-NOAA Here is a run directory from workflow provided by @aerorahul:

/scratch1/NCEPDEV/stmp2/Jun.Wang/esmfthrd/clean_test

The corresponding UFS RT test is: cpld_control_noaero_p8. Please check the differences in input.nml, model.configure, nems.configure, diag_table, field_table. Thanks

@aerorahul Would you please provide a fully coupled test case with GOCART? Thanks

**Comment by DeniseWorthen** [comment]
There is additional work to be done for the ice_in namelist.

**Comment by zach1221** [comment]
> There is additional work to be done for the ice_in namelist.

Hi, @DeniseWorthen . following up here to see if this enhancement should remain open?

**Comment by DeniseWorthen** [comment]
@zach1221 I have not been working on this, and I don't know what the current status is. I don't even remember what the issue was with the ice_in. It was, after all, 10 months ago?

**Comment by zach1221** [comment]
> @zach1221 I have not been working on this, and I don't know what the current status is. I don't even remember what the issue was with the ice_in. It was, after all, 10 months ago?

I don't think it was an issue so much as an enhancement, based on the label. Was just checking to see what additional work was needed to be done, based on your comment above, when the ticket was re-opened.

Copying @junwang-noaa, since she is the owner. Jun, should this be closed? PR [#1699](https://github.com/ufs-community/ufs-weather-model/pull/1699) was merged, and that PR appears to say this enhancement could be closed with its merging.

**Comment by junwang-noaa** [comment]
I think this can be closed when Dan's PR[#2425](https://github.com/ufs-community/ufs-weather-model/pull/2425) (in the queue) and G-W PR[#3017](https://github.com/NOAA-EMC/global-workflow/pull/3017) are committed.

**Comment by gspetro-NOAA** [comment]
Both of the listed G-W PRs have been committed. Closing.
